### DISCUSSION TITLE
Anybody out there with a trained Yolov5x on the Crowdhuman dataset?

### DISCUSSION THREAD

**Discussion by mikel-brostrom** [ultralytics/yolov5#8165] [Q&A]
Hi!

Is there anybody out there with a Yolov5x trained on the crowdhuman dataset (and ideally fine-tuned
either on MOT16 or 17). I want  to evaluate my tracking solution (https://github.com/mikel-brostrom/Yolov5_StrongSORT_OSNet) but have been stuck with a Yolov5m, trained on the crowdhuman dataset only, that I trained a while back. Sadly, none of that code is left and I am mostly focusing on the tracking part which takes all of my time. The tracker (https://github.com/mikel-brostrom/Yolov5_StrongSORT_OSNet/wiki/Evaluation) performs quite well given the Yolo model used for generating the detections, scoring top18 on HOTA and achieving SOTA results on ID switches. I am quite confident that a top 5 HOTA result could be achieved with a Yolov5x model. If there is any kind soul out there who happens to have a Yolov5x train on the crowdhuman dataset or could help out training one, reach me out at [yolov5.deepsort.pytorch@gmail.com](mailto:yolov5.deepsort.pytorch@gmail.com)

**Comment by glenn-jocher** [comment]
@mikel-brostrom hi, thank you for your feature suggestion on how to improve YOLOv5 🚀!

Crowdhuman isn't part of our autodownload datasets, but if you've used it before please consider submitting a Pull Request to help future users more easily train YOLOv5 on crowdhuman. 

https://github.com/ultralytics/yolov5/tree/master/data
<img width="1012" alt="Screen Shot 2022-06-10 at 3 06 10 PM" src="https://user-images.githubusercontent.com/26833433/173070753-424b8d7c-db37-4dcb-bb8f-e9cd6531c690.png">

Please see our ✅ [Contributing Guide](https://github.com/ultralytics/yolov5/blob/master/CONTRIBUTING.md) to get started.
